Common/Source/Comm/device.cpp

```diff
@@ -599,18 +629,21 @@ BOOL devInit() {
             ReadPortSettings(i, Port, &SpeedIndex, &BitIndex);
         }
         // remember: Port1 is the port used by device A, port1 may be Com3 or Com1 etc
+        /*
```

Comment:
useless comment, does not help to understand code ....

Reply:
removed